Update zh_cn for 1.20

ChuijkYahus opened the pull request:
This PR is not urgent error fixes, it currently only consists of typo fixes and readability fixes.
If new commits to `en_us` may occur in the future, they would be updated here asap.

ChuijkYahus committed:
update

ChuijkYahus committed:
wording

ChuijkYahus committed:
wording again

ChuijkYahus committed:
Merge branch 'Alexthw46:1.20.1-Neo' into 1.20.1-Neo

ChuijkYahus committed:
zh_cn lang update

ChuijkYahus committed:
Merge branch 'Alexthw46:1.20.1-Neo' into 1.20.1-Neo

ChuijkYahus committed:
zh_cn lang fix: correcting and polishing

ChuijkYahus commented:
The five new plants and fungi's translations are still subject to change. The current ones aren't the final result.

Alexthw46 commented:
Last changes to codex should allow to automatically split long lang entries into multiple pages. It's all based on the translation key but the reload system doesn't work well so at best it needs to reload lang and go back to main screen, at worst to launch the game again.

Lang-wise, there should only be a last batch of text to be added about the scriptorium and the incenses when they're finalized

ChuijkYahus commented:
Thanks a lot!

The spaces are there mainly to deal with word-wrap problems. The published versions would display the whole key in one line, regardless of page limits. Going to test whether spaces are still needed.

Edit: They are still needed.

skyraah commented:
提交到此处以防群聊消息遗漏
Discussions about translations are recorded here
---
|原名|译名（Chuijk ver.）|译名（Skyraah ver.）|
|-|-|-|
|Flowering Oanna|盛开的欧安那莲|盛绽净莲/盛绽异莲|
|Merammer Root|梅拉莫根茎|/|
|Sildrian Seed|森泽亚种子|蕴生林种/蕴生之种|
|Sprig of Avenna|阿温那蕨小枝|阿梵娜嫩枝/萦缠卷芽/萦缠蕨（枝）|

## 翻译参考

### Oanna

参考 #86 

### Sildrian
参考了物品描述
```json
{
"lore.eidolon.sildrian_seed": "Restless energy within."
}
```

此处Chuijk翻译为`蕴含着涌动的能量。`

又及 #86 处讨论
> - Sildrian
个人推断和“sylvan”，或者拉丁文的“silva”/“森林”有所联系
这个打算在音译的过程中塞个“森”进去

故此翻译为`蕴生林种`，`蕴生之种`可作为备选

### Avenna

#### 阿梵娜嫩枝

重新音译，且根据[剑桥词典](https://dictionary.cambridge.org/zhs/%E8%AF%8D%E5%85%B8/%E8%8B%B1%E8%AF%AD-%E6%B1%89%E8%AF%AD-%E7%AE%80%E4%BD%93/sprig)
 > 【sprig】
> - a single small plant stem with leaves on it
> - 带叶小枝;嫩枝

将`小枝`调整为`嫩枝`

#### 萦缠卷芽/萦缠蕨（枝）

来源于其描述：
> 紧缠不放。
> Won’t let go.

取其中缠绕含义，引用萦缠一词
将`sprig`引申为嫩芽，结合贴图形态，取`卷芽`一词

`蕨（枝）`取自原译版

ChuijkYahus committed:
zh_cn lang fix: Oanna, Sildrian, Avenna

Co-Authored-By: cyciling <1377899390@qq.com>

skyraah commented:
Merammer Root有关讨论
Merammer Root Translation discus
---
|原名|译名（Chuijk ver.）|译名（Skyraah ver.）|
|-|-|-|
|Merammer Root|梅拉莫根茎|净萃根茎|
|Merammer Resin|梅拉莫凝脂|净萃凝脂|

## 翻译参考
### Merammer
由用途推断而来，结合一译`纯净块茎`，取“纯净”与“萃取”二词生造

ChuijkYahus committed:
zh_cn lang fix: Merammer

Co-Authored-By: cyciling <1377899390@qq.com>

ChuijkYahus committed:
zh_cn lang fix: polishing

This was to polish names for the four plants, the main goal is to reduce the use of overly complicated words and to make them easy to remember.
For more info, please check the comments in related PRs.

Co-Authored-By: cyciling <1377899390@qq.com>

ChuijkYahus commented:
This table here is just to record the thoughts on plants' names.

|English Name|Current Chinese Trsl.|Memo|
|:-:|:-:|-|
|Flowering Oanna|盛放净莲|"净莲"(Oanna, *lit.* pure lotus/waterlily)<br/>The same as comments above, tweaked slightly for common word choice|
|Merammer Root|腴须根茎|"腴须"(Merammer, *lit.* plump (usually branching) roots)<br/>"腴" also has the meaning of "fertile"<br/>To better describe its shape and refer to its lore|
|Sildrian Seed|繁林之种|"繁" means "lush" here<br/>The same as comments above, tweaked for conciseness|
|Sprig of Avenna|卷蕨小枝|"卷蕨"(Avenna, *lit.* curled fern)<br/>To better distinguish "Avenna" and "Sprig"|

ChuijkYahus committed:
Merge branch 'Alexthw46:1.20.1-Neo' into 1.20.1-Neo

ChuijkYahus committed:
Merge branch 'Alexthw46:1.20.1-Neo' into 1.20.1-Neo

ChuijkYahus committed:
zh_cn lang update

Alexthw46 commented:
While there might still be changes incoming, it's been too much time and i should roll playtest of the new stuff while i debug the 1.21 port.
I'm merging this, it will appear on 
https://maven.blamejared.com/com/alexthw/eidolon_repraised/eidolon_repraised-1.20.1/
so the translation can be tested too.

In the next days i'll post on the discord., I'm not 100% convinced on some things of the update so it will approach the curse/rinth sites only after a feedback roll
